packages/fiori/src/i18n/messagebundle.properties

```diff
@@ -44,9 +41,18 @@ NOTIFICATION_LIST_ITEM_SHOW_LESS=Less
 #XBUT: Tooltip text for 'Overflow' button in the NotificationListItem, that opens a menu with actions
 NOTIFICATION_LIST_ITEM_MENU_BTN_TITLE=Actions
 
+#XBUT: Label text for 'More' lin in the NotificationListItem, that shows the full text, if truncated
+NOTIFICATION_LIST_ITEM_MORE_LINK_LABEL_FULL=More button. Show the full texts
```

Comment:
the button shows only one text for the respective item. I suggest:
...Show the full text

Reply:
The "button" show the full text of the title and full text of the description, so they are two texts
![image](https://github.com/SAP/ui5-webcomponents/assets/7242508/fdd2c007-9cc8-48c3-b689-e2f913a6c4e3)